indico/htdocs/js/indico/RegistrationForm/registrationform.js

```diff
@@ -205,11 +209,14 @@ ndRegForm.directive('ndRegForm', function($rootScope, url, sortableoptions, regF
                 removeSection: function(section) {
                     regFormFactory.Sections.remove({
                         confId: $rootScope.confId,
-                        sectionId: section.id
-                    }, {}, function(updatedSections) {
-                        regFormFactory.processResponse(updatedSections, {
-                            success: function(updatedSections)  {
-                                $scope.sections = updatedSections;
+                        sectionId: section.id,
+                        confFormId: $rootScope.confFormId
+                    }, {}, function(data) {
+                        regFormFactory.processResponse(data, {
+                            success: function()  {
+                                $scope.sections = $scope.sections.filter(function(obj) {
```

Comment:
So now we are not nuking the view in the client with that the server sends? Also, `data` is not used?